hieradata/common.yaml

```diff
@@ -1561,6 +1561,7 @@ hosts::production::management::hosts:
     legacy_aliases:
       - 'monitoring'
       - "grafana.%{hiera('app_domain')}"
+      - "alert.%{hiera('app_domain')}"
```

Comment:
Adding a new "legacy" alias seems a bit not-nice, but the other solution is disabling certificate veriication... this is probably the lesser evil.

Reply:
Yeah, I couldn't think of a better solution... makes me sad too :cry: